third_party/Makefile

```diff
@@ -178,17 +178,19 @@ directx_amf_encoding_latency:
 # Install Megatron-LM
 megatron_lm:
 	if [ ! -d "Megatron/Megatron-LM" ]; then \
-        git clone "https://github.com/NVIDIA/Megatron-LM.git" "Megatron/Megatron-LM"; \
-    fi
+        	git clone -b 52b7a18 "https://github.com/NVIDIA/Megatron-LM.git" "Megatron/Megatron-LM"; \
```

Comment:
Should we need to add a submodule? @abuccts

Reply:
it's ok to add

Reply:
@abuccts, do you know how to solve this issue, when git clone submodule, it saids there's filename too long
![image](https://github.com/microsoft/superbenchmark/assets/37182275/be9fe859-db47-4cb4-aca4-5edd71ea0f99)

Reply:
try https://stackoverflow.com/a/22575737/7630831